.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+
+on: [push, pull_request]
+
+env:
+  DISPLAY: :99
+  IS_PULLREQUEST: ${{ github.event_name == 'pull_request' }}
```

Comment:
This should work fine (returns "false")

https://github.com/andy5995/appimage.github.io/runs/3002651747?check_suite_focus=true#step:2:13